starter-kits.json

```diff
@@ -6,6 +6,7 @@
   "expo-zustand-styled-component": "Expo, Zustand, and Styled Components",
   "express-apollo-prisma": "Express.js, Apollo Server, and Prisma",
   "express-typeorm-postgres": "Express.js, TypeOrm, and PostgreSQL",
+  "lit-mobx-vaadin": "LitElement, Mobx, and Vaadin",
```

Comment:
Lit or LitElement?

Reply:
I changed all occurrences of LitElement to Lit 😄